include/snowhouse/stringizers.h

```diff
@@ -48,7 +48,7 @@ namespace snowhouse
       template<typename>
       static no compile_time_check_const_iterator(...);
 
-      static const bool value = sizeof(compile_time_check_const_iterator<T>(0)) == sizeof(yes);
+      static const bool value = sizeof(compile_time_check_const_iterator<T>(nullptr)) == sizeof(yes);
```

Comment:
Snowhouse currently supports older C++ versions than C++11. (This is going to change with the next major release...but I haven't had time for that.) Please use the `#ifdef SNOWHOUSE_HAS_NULLPTR` guard for the change.

Reply:
I did by duplicating the line. Is it ok for your or you prefer me to define a specific `SNOWHOUSE_HAS_NULLPTR` definition?

Reply:
Like:

```cpp
#ifdef SNOWHOUSE_HAS_NULLPTR
#define SNOWHOUSE_NULLPTR nullptr
#else
#define SNOWHOUSE_NULLPTR 0
#endif
```

Reply:
Let's keep it this way. Thanks.